src/client/ue_worker.cc

```diff
@@ -285,21 +283,25 @@ void UeWorker::DoFftData(size_t tag) {
 
   const size_t sig_offset = config_.OfdmRxZeroPrefixClient();
   const size_t dl_symbol_id = config_.Frame().GetDLSymbolIdx(symbol_id);
+  const size_t dl_data_symbol_id = dl_symbol_id -
+                                   config_.Frame().ClientDlPilotSymbols();
 
   if (kRecordDownlinkFrame) {
-    if (frame_id == kRecordFrameIndex) {
-      const std::string short_id =
+    if (frame_id > 0 && frame_id % kRecordFrameInterval == 0) {
+      const std::string short_serial =
           config_.UeRadioId().empty()
-              ? ""
-              : "_" + config_.UeRadioId().at(0).substr(
-                          config_.UeRadioId().at(0).length() - kShortIdLen);
-      std::string fname = "rxdata" + std::to_string(dl_symbol_id) + short_id +
-                          "_" + std::to_string(ant_id) + ".bin";
+              ? "UE"
+              : config_.UeRadioId().at(ant_id).substr(
+                    config_.UeRadioId().at(ant_id).length() - kShortSerialLen);
+      const std::string fname_ext = std::to_string(frame_id) + "_" +
```

Comment:
For smaller intervals, this will generate too many files. Perhaps we should record in a subdirectory.